Qualification/input/qualification_plan.json

```diff
@@ -0,0 +1,575 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/QualificationPlan/v3.1/schemas/OSP_Qualification_Plan_Schema.json",
+    "Projects": [
+        {
+            "Id": "Probenecid",
+            "Path": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Probenecid-Model/refs/heads/main/Probenecid-Model.json"
+        },
+        {
+            "Id": "Furosemide",
+            "Path": "https://raw.githubusercontent.com/mgsc1020/Furosemide-Model/refs/heads/main/Furosemide-Model.json"
+        },
+        {
+            "Id": "Probenecid-Furosemide-DDI",
+            "Path": "../../Probenecid-Furosemide-DDI.json",
+            "BuildingBlocks": [
+                {
+                    "Type": "Compound",
+                    "Name": "Probenecid",
+                    "Project": "Probenecid"
+                },
+                {
+                    "Type": "Formulation",
+                    "Name": "tablet",
+                    "Project": "Probenecid"
+                },
+                {
+                    "Type": "Compound",
+                    "Name": "Furosemide",
+                    "Project": "Furosemide"
+                }, 
+                {
+                    "Type": "Formulation",
+                    "Name": "Solution",
+                    "Project": "Furosemide"
+                },
+                {
+                    "Type": "Formulation",
+                    "Name": "Immediate release",
+                    "Project": "Furosemide"
+                }
+
+            ]
+        }
+    ],
+    "ObservedDataSets": [
+        {
+            "Id": "DDI Ratios",
+            "Path": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Database-for-observed-data/refs/heads/master/DDI.csv",
```

Comment:
@mgsc1020 Update here to

"https://raw.githubusercontent.com/Open-Systems-Pharmacology/Database-for-observed-data/v1.5/DDI.csv"